tutorials/tcpip/pcap-driver/main.c

```diff
@@ -165,6 +165,8 @@ static void http_ev_handler(struct mg_connection *c, int ev, void *ev_data) {
   (void) ev_data;
 }
 
+static struct mg_connection *mqtt_conn = NULL;
```

Comment:
We need to retry the connection because DNS resolution will fail, as the first UDP datagram is always lost when sending to local machines (that includes the router) and DHCP is not enabled (because when having DHCP enabled we first assume the gateway MAC is the DHCP server MAC, hence DNS resolution works